### PR TITLE
add support for AppCode 3.1

### PR DESCRIPTION
**Pull request by morphinewan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
 
 - Remove the scripts dir from Messages syncing (@vitorgalvao)
+- Added support of AppCode 3.1 (via @morphinewan)
 
 ## Mackup 0.8.2
 
 - Fix: Users with a list of apps to sync were getting an error on restore or
   uninstall
 
+
```
Comment (lra) [inline CHANGELOG.md]: why? =)

File: mackup/applications/appcode-31.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = AppCode 3.1
+
+[configuration_files]
+Library/Application Support/appCode31
+Library/Preferences/appCode31
```
Comment (lra) [inline mackup/applications/appcode-31.cfg]: every line should end with a newline character

Comment (morphinewan) [inline mackup/applications/appcode-31.cfg]: @lra It seems github online view mode hide the last newline character automaticlly. I can see the newline character in edit mode.

**Comment by morphinewan** [comment]
Sorry for the newline character being messed up.

**Comment by lra** [comment]
thanks

**Pull request merged**